### PR TITLE
8276036: The value of full_count in the message of insufficient codecache is wrong

### PR DESCRIPTION
**Pull request by tkiriyama**
Could you please review the 8276036 bug fixes?

This bug is caused by the wrong place to add the value of full_count.
The initial value of full_count is 0, so it needs to be added before outputting the message.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [ ] Change must be properly reviewed

### Issue
 * [JDK-8276036](https://bugs.openjdk.java.net/browse/JDK-8276036): The value of full_count in the message of insufficient codecache is wrong ⚠️ Issue is not open.

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/6129/head:pull/6129` \
`$ git checkout pull/6129`

Update a local copy of the PR: \
`$ git checkout pull/6129` \
`$ git pull https://git.openjdk.java.net/jdk pull/6129/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 6129`

View PR using the GUI difftool: \
`$ git pr show -t 6129`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/6129.diff">https://git.openjdk.java.net/jdk/pull/6129.diff</a>

</details>

### CODE REVIEW

File: test/hotspot/jtreg/compiler/codecache/CodeCacheFullCountTest.java [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+    }
+
+    public static void runtest() throws Throwable {
+        ProcessBuilder pb = ProcessTools.createJavaProcessBuilder(
+          "-XX:ReservedCodeCacheSize=2496k", "-XX:-UseCodeCacheFlushing", "CodeCacheFullCountTest", "WasteCodeCache");
+        Process javaProcess = pb.start();
+        javaProcess.waitFor();
+
+        OutputAnalyzer javaOutput = new OutputAnalyzer(javaProcess);
+        String stdout = javaOutput.getStdout();
```
Comment (chhagedorn) [inline test/hotspot/jtreg/compiler/codecache/CodeCacheFullCountTest.java:63]: Use `ProcessTools.executeProcess(pb)` instead:
```
OutputAnalyzer oa = ProcessTools.executeProcess(pb);
oa.shouldHaveExitValue(0);
String stdout = oa.getStdout();
```

**Comment by chhagedorn** [comment]
Hi @tkiriyama, this bug was already fixed and integrated by @tobiasholenstein a few days a ago:
https://github.com/openjdk/jdk/pull/6185
https://github.com/openjdk/jdk/commit/61cb4bc6b0252536364a86f38ff2e5c8c7ab610b

Since your account was only verified yesterday, an RFR email could not be sent out to the email list to inform people about your fix. Please make sure that you always assign JBS issues to you when you intend to work on them (the issue was assigned to @tobiasholenstein). If you do not have an account, reach out to someone with an account to reserve/sponsor it for you. This avoids duplicated work. 

Nevertheless, it is nice that you also found a test to verify your fix. I suggest to file a follow-up RFE to add your test for JDK-8276036 separately.

Thanks,
Christian

**Comment by tkiriyama** [comment]
Hi, @chhagedorn , thank you for your valuable comments. I will try that.

**Comment by tkiriyama** [comment]
JDK-8276036 was already fixed. I close this pull request.

**Pull request closed without merging**